src/Arcus.Observability.Tests.Integration/appsettings.json

```diff
@@ -1,4 +1,7 @@
 ﻿{
   "Arcus": {
+    "ApplicationInsights": {
+      "InstrumentationKey": "#{Arcus.ApplicationInsights.InstrumentationKey}#" 
```

Comment:
1. Did you add it as a secret in variable group?
2. How about `Arcus.ApplicationInsights.InstrumentationKey`? Don't think we prefix our other variable names as well

Reply:
1. I reused the web Api secret group
2. I don't see what you mean? Your suggestion is the same as it is now?

Reply:
1. Is that because the secret is already there or not?
2. Ah yes, I meant without `Arcus.` 😂

Reply:
1. Yes, instrumentation key is already defined there
2. The instrumentation key was named this way in that group.

Reply:
I'd suggest to create a new Application Insights instance to reduce the noise during troubleshooting for these tests, are you up for that?

I would then use `ApplicationInsights.InstrumentationKey` from an "Observability" variable group.

Reply:
Ok, I'll do that.

Reply:
Thanks and sorry for the hassle!

Reply:
All done now?